crates/starknet-devnet-core/src/starknet/get_class_impls.rs

```diff
@@ -18,11 +18,7 @@ pub fn get_class_hash_at_impl(
     let core_address = contract_address.try_into()?;
 
     let class_hash = state.get_class_hash_at(core_address)?;
```

Comment:
The state no longer needs to be mutable. Perhaps we can deal with this in a separate issue/PR.